8.0/BlazorWebAssemblyStandaloneWithIdentity/Backend/Program.cs

```diff
@@ -38,6 +41,15 @@
 
 var app = builder.Build();
 
+#if DEBUG
+using (var scope = app.Services.CreateScope())
+{
+    var services = scope.ServiceProvider;
+
+    SeedData.Initialize(services);
+}
+#endif
```

Comment:
Some people use debug builds in production to get line numbers for stack traces and things like that. It's better to check `IsDevelopment()`.

```suggestion
if (builder.Environment.IsDevelopment())
{
    using (var scope = app.Services.CreateScope())
    {
        var services = scope.ServiceProvider;
    
        await SeedData.InitializeAsync(services);
    }
}
```